assignments/assignment01.rb

```diff
@@ -80,8 +78,15 @@ def my_reverse(s)
 # Write a method `palindrome?`
 # that determines whether a string is a palindrome
 def palindrome?(s)
-  stripped = s.delete(" ").delete(",").downcase
-  stripped == stripped.reverse
+  s1 = s.sub(' ', '').downcase
+  s2 = s1.reverse
+
+  if s2 == s1 
```

Comment:
You don't need to do an if statement here. Your goal is to return true or false, depending on whether s2 == s1. The line "s2 == s1" will do exactly that.